Jint/Native/Number/Dtoa/CachePowers.cs

```diff
@@ -52,17 +51,16 @@ internal CachedPower(ulong significand, short binaryExponent, short decimalExpon
         }
 
 
-        internal static int GetCachedPower(int e, int alpha, int gamma, DiyFp cMk)
+        internal static (short, DiyFp) GetCachedPower(int e, int alpha, int gamma)
```

Comment:
Would there be less allocations if you used two `out` values instead of a `Tuple` ?

Reply:
ValueTuple is a struct so it wont' allocate memory. I did quick check on the out alternative and benchmarkdotnet showed both ways to allocate 0B and ValueTuple version being slightly faster (can't explain that).